Makefile.manual

```diff
@@ -331,6 +331,31 @@ datasets/wikidatawiki.labeled_revisions.w_cache.20k_2015.json: \
 		--host https://wikidata.org \
 		--verbose > $@
 
+datasets/wikidatawiki.sampled_revisions.500k_2018.json:
+	wget -qO- https://quarry.wmflabs.org/run/271461/output/0/json-lines?download=true > $@
+
+datasets/wikidatawiki.autolabeled_revisions.500k_2018.json: \
+		datasets/wikidatawiki.sampled_revisions.500k_2018.json
+	cat $< | \
+	./utility autolabel --host=https://www.wikidata.org \
+		--trusted-groups=bot,sysop,bureaucrat \
+		--trusted-edits=1000 \
+		--revert-radius=3 \
+		--revert-window=48 \
+		--verbose > $@
+
+datasets/wikidatawiki.autolabeled_revisions.500k_2018.no_review.json: \
+		datasets/wikidatawiki.autolabeled_revisions.500k_2018.json
+	cat $< | grep -E '"needs_review": (false|"False")' > $@
+
+datasets/wikidatawiki.autolabeled_revisions.500k_2018.review.json: \
+		datasets/wikidatawiki.autolabeled_revisions.500k_2018.json
+	cat $< | grep -E '"needs_review": (true|"True")' > $@
+
+datasets/wikidatawiki.revisions_for_review.5k_2018.json: \
+		 datasets/wikidatawiki.autolabeled_revisions.500k_2018.review.json
+	cat $< | shuf -r -n 5000 > $@
```

Comment:
Are there substantially more than 5k?  Also the -r is certainly unwanted.  It means to shuffle with replacement, so if there are less than 5k, there will be duplicates.

Reply:
They are 20K

Reply:
We should cut the initial sample down by a factor of 4 then to get to ~5k.  So 125k sampled revisions --> autolabel --> 5k needs review.  Does that make sense?